fivetran_provider_async/operators.py

```diff
@@ -48,6 +50,7 @@ class FivetranOperator(BaseOperator):
     :param poll_frequency: Time in seconds that the job should wait in between each try.
     :param reschedule_wait_time: Optional, if connector is in reset state,
             number of seconds to wait before restarting the sync.
+    :param wait_for_completion: Wait for Fivetran sync to complete to finish the task.
     :param deferrable: Run operator in deferrable mode. Default is True.
```

Comment:
nitpick: would be better if we could keep the order consistent

```suggestion
    :param deferrable: Run operator in deferrable mode. Default is True.
    :param wait_for_completion: Wait for Fivetran sync to complete to finish the task.
```